app.py

```diff
@@ -229,12 +229,33 @@ def user_signup():
     password = request.json['password']
     email=request.json['email']
 
-    new_user = Users(username, password, email)
+    email_check = Users.query.filter_by(email=email).first()
 
+    if email_check:
+        return jsonify({
+            'username':'',
+            'password':'',
+            'email':'',
+            'admin_status':0,
+            'jwt':'',
+            'status':'User Already Exsists'
```

Comment:
Use enums for statuses

Reply:
Yesss, it’s in todo. Will make proper responses.